bindings/python/iota_sdk/client/_node_indexer_api.py

```diff
@@ -110,6 +113,22 @@ def __init__(self, dict: Dict):
             self.items = [OutputId.from_string(
                 output_id) for output_id in dict["items"]]
 
+    def output_ids(
+            self, query_parameters: QueryParameters) -> OutputIdsResponse:
+        """Fetch alias/basic/NFT output IDs from the given query parameters.
+        Supported query parameters are: "hasNativeTokens", "minNativeTokenCount", "maxNativeTokenCount", "unlockableByAddress", "createdBefore", "createdAfter".
```

Comment:
Incomplete list ?

Reply:
Or we just never put Cursor/PageSize ?

Reply:
Copied from the plugin source :trollface: